### ISSUE TITLE
Build issue error[E0277]: can't compare

### ISSUE THREAD

**Issue by presscot** [cloudflare/quiche#699]
Commit: 5331a2758a29e29e97a3cd7191eba3330f0126e1
File: src/lib.rs 
Line: 1479

if hdr.dcid.as_ref() != self.scid  ^^ no implementation for `&[u8] == std::vec::Vec<u8>`

**Comment by junhochoi** [comment]
I don't have same issue rust 1.46.0 and rust 1.47.0. If you use older version of rust, please try with newer rustc.

**Comment by jaydouble** [comment]
ubuntu 20.04 is shipped with rustc version 1.43 and that is failing, after removing rustc/cargo, and than installing with:
```
curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh
```
it is now compiling without error

**Comment by ghedo** [comment]
Glad to hear your problem is fixed! Given that we already document the minimum Rust version required in README.md I don't think there's anything else to do here, so closing.
